### ISSUE TITLE
Add ability to pull player stats

### ISSUE THREAD

**Issue by seannybgoode** [the-hideout/tarkov-api#268]
Looking to develop an app that relies on player stats, looking to see if tarkov-api can do this.

**Comment by Shebuka** [comment]
There are no player stats in the tarkov.dev API

**Comment by Razzmatazzz** [comment]
What kind of stats are you interested in?

**Comment by seannybgoode** [comment]
> What kind of stats are you interested in?

Kills, Deaths, Survival Rate, Raid Survival Time, Account Lifetime. Flea market rep would be a good one too.

**Comment by tomonari-t** [comment]
I would also be happy if I could get stats for each player.
I would like to create an app that can track changes in K/D, money, etc.

**Comment by mrdeadlocked** [comment]
This would be great.

**Comment by Razzmatazzz** [comment]
The player search endpoints used by the Tarkov.dev website are protected by Cloudflare Turnstile to prevent automated requests. This is necessary to prevent our upstream data provider from getting overwhelmed with requests.

However, all profiles that are viewed on Tarkov.dev are available at `https://players.tarkov.dev/profile/[accountId].json`. An index of all retrieved account ids and player names is available at `https://players.tarkov.dev/profile/index.json`. An index of account ids and the timestamp for when that profile was last updated is available at `https://players.tarkov.dev/profile/updated.json`. For PVE profiles, just replace `/profile/` in those paths with `/pve/`. The index files are updated once per day.

**Comment by mrdeadlocked** [comment]
Would it be possible to index all users once daily?

**Comment by Razzmatazzz** [comment]
Not really. With a conservative estimate of 2.8 million accounts, that would be 116,666.67 profile queries per hour / 1,944.44 per minute / 32.4 per second. Sending that volume of requests just isn't feasible.
